# Yellow Trip Data

Este dataset é um conjunto de informações relacionadas as viagens de Táxis Amarelos na cidade de Nova York em Abril de 2019. Para as séries temporais extraidas a partir deste dataset os registros temporais são sempre o momento de início da viagem. A partir dos dados foram extraidas as seguintes tuplas para séries temporais:

- Registro Temporal - Quantidade de Passageiros
- Registro Temporal - Total Pago
- Registro Temporal - Distância

A seguir algumas considerações sobre esses dados.

In [2]:
(2173865 * 16) /15.5834

2231980.1840419937

## Depois de executada a compressão os seguintes resultados foram obtidos:

| Atribudo            | Tempo de Compressão      | Taxa de Compressão | Velocidade de Compressão |
| ------------------- | -----------------------: | -----------------: | -----------------------: |
| passager_count      | 7.8422                   | 10.8615            | 4442996.0980             |
| trip_distance       | 16.0706                  | 1.8971             | 2168112.2048             |
| total_amount        | 15.5834                  | 2.1332             | 2231980.1840             |